src/utils/utils.go

```diff
@@ -41,6 +42,24 @@ func InitLogger(verbose bool) *logrus.Logger {
 	return log
 }
 
+type CoreosInstallerLogWriter struct {
+	progressRegex *regexp.Regexp
+	log           *logrus.Logger
+	lastLogLine   []byte
+}
+
+func NewCoreosInstallerLogWriter(logger *logrus.Logger) *CoreosInstallerLogWriter {
+	re := regexp.MustCompile(`^>(.*?)\((.*?)\)\s*\r`)
+	return &CoreosInstallerLogWriter{re, logger, []byte{}}
+}
+
+func (l *CoreosInstallerLogWriter) Write(p []byte) (n int, err error) {
+	if l.progressRegex.Match(p) {
+		l.log.Info(string(p))
+	}
+	return len(p), nil
```

Comment:
Don't we need to add these bytes to the lastLogLine?

Reply:
I'll add it once it's required.